code/modules/hydroponics/grown/nettle.dm

```diff
@@ -50,12 +50,12 @@
 /obj/item/grown/nettle/pickup(mob/living/user)
 	..()
 	if(!ishuman(user))
-		return FALSE
+		return TRUE
```

Comment:
Wait, this would allow non-human to pick it up.

Probably not a good idea since the next line immediately assume they're a human